src/libs/PolicyUtils.js

```diff
@@ -57,6 +57,42 @@ function hasCustomUnitsError(policy) {
     return !_.isEmpty(_.pick(lodashGet(policy, 'customUnits', {}), 'errors'));
 }
 
+/**
+ * @param {Number} value
+ * @param {Function} toLocaleDigit
+ * @returns {Number}
+ */
+function getNumericValue(value, toLocaleDigit) {
+    const numValue = parseFloat(value.toString().replace(toLocaleDigit('.'), '.'));
+    if (Number.isNaN(numValue)) {
+        return NaN;
+    }
+    return numValue.toFixed(3);
```

Comment:
This `3` looks like a magic number - maybe it would be worth to extract it to `CONST.js`?